kratos/python_scripts/run_tests.py

```diff
@@ -165,7 +165,7 @@ def RunCppTests(self, applications):
             import_module("KratosMultiphysics." + application)
 
         try:
-            KtsMp.Tester.SetVerbosity(KtsMp.Tester.Verbosity.PROGRESS)
+            KtsMp.Tester.SetVerbosity(KtsMp.Tester.Verbosity.TESTS_OUTPUTS)
```

Comment:
please don't change this or make a separate PR if you think this should be changed

Reply:
sure. I thought I would have reverted it. Thx.